CHANGES.md

```diff
@@ -58,6 +58,7 @@
 ## Breaking Changes
 
 * HBaseIO.ReadAll now requires a PCollection of HBaseIO.Read objects instead of HBaseQuery objects ([BEAM-9279](https://issues.apache.org/jira/browse/BEAM-9279)).
+* ProcessContext.updateWatermark has been removed in favor of using a WatermarkEstimator ([BEAM-9430](https://issues.apache.org/jira/browse/BEAM-9430)).
```

Comment:
Do we plan to add some end-user documentation for WatermarkEstimators or SDF in general?

Reply:
Oh sorry it was just obviously good so I proceeded. Next time I will wait.

Reply:
Yes, there should be some examples of how to write SDFs added to our docs.
After Java SDFs have a way to report progress and backlog, then we should be able to do this in Python and Java (and mention that Go is in the works).

Are the docs something you were interested in doing?

Reply:
Interested yes but unfortunately won't have time to work on that at the moment.